pkg/apis/options/load_test.go

```diff
@@ -413,7 +416,13 @@ sub:
 				} else {
 					Expect(err).ToNot(HaveOccurred())
 				}
+
 				Expect(input).To(Equal(in.expectedOutput))
+
+				// Unset the environment variables after running the test
+
+				os.Unsetenv("TESTUSER")
```

Comment:
Can you `defer` this rather than setting it here, would be easier to not forget it that way and make sure it's unset on every exit

Reply:
done